server/sonar-db-dao/src/main/resources/org/sonar/db/component/ComponentMapper.xml

```diff
@@ -340,7 +340,7 @@
   </sql>
 
   <select id="selectUuidsForQualifiers" resultType="UuidWithProjectUuid">
-    SELECT p.uuid as "uuid", p.project_uuid as "projectUuid" FROM projects p
+    SELECT p.uuid, p.project_uuid as "projectUuid" FROM projects p
```

Comment:
restore this mapping, as the result is mapped to a Bean. Mapping is used to map to a bean of the property. It should be kept.